### ISSUE TITLE
Past Appointment Details - Accessibility Issues

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#17003]
## What Happened?
NOTE: Adding that this bug also applies to Canceled Appointments. -Ken

**VoiceOver / TalkBack**

1. With the appropriate screen reader enabled on iOS and Android, the a11yHint for the back button is not being picked up 
     - Note: affects all past appointments (VA appointment, VAVC at home; etc)
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)
2. With TalkBack enabled on Android, the "...visit VA.gov" text is incorrectly picked up/pronounced
     - Note: affects all past appointments (VA appointment, VAVC at home; etc)

## Specs:
- Device: Android -Google Pixel 3XL(v.11) with TalkBack enabled
- Device: iOS - iPhone XR (v.14.2) with VoiceOver enabled
- User: marcy.nadeau@va.gov

## Steps to Reproduce
**VoiceOver / TalkBak**

For 2:
1. With TalkBack enabled on Android, navigate to a details page of a past appointment
2. Select the "...visit VA.gov" text in the "schedule another appointment" section
3. Note that VA.gov is picked up/pronounced as Vaw [dot] gov 

## Desired Behavior
**VoiceOver / TalkBak**

For 2:
- With TalkBack enabled on Android, the "...visit VA.gov" text will be picked up/pronounced as VA [dot] gov

## Acceptance Criteria
- See desired behavior

## Bug Severity
- 4 - Low

## Linked Story (s)
- Story #15754
- and #15739

## Screen Shot (s) and Additional Information
2. Showing the affected areas 
![Past Appointment Details - Android Screen Reader.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/039505c2-aa21-4a54-897b-fbbb9731e1d6)

**Comment by digitalken** [comment]
PASS 12/16/2020 - VA.gov is no read correctly.
